docs/protocol/governance/create-proposal.md

```diff
@@ -75,44 +147,110 @@ cd cgp-(your proposal number)
+- **Presentation Slot**: Be prepared to present your proposal succinctly. Typically, you may be allocated a specific time slot, such as 5 minutes for presentation and 5 minutes for Q&A.
+- **Technical Questions**: Expect technical questions from the community, especially if your proposal involves code changes. Be ready to explain complex concepts in accessible language.
+- **Community Feedback**: Governance calls are an opportunity for the community to provide direct feedback. Take notes and be open to incorporating this feedback into your proposal.
+- **Approval Indicators**: Use these calls as a temperature check on your proposal's likelihood of passing. Positive engagement and constructive feedback are good indicators.
+- **Networking**: These calls are an excellent opportunity to network with other members of the Celo ecosystem, which can be valuable for building future support.
 
-```bash
-celocli governance:execute --proposalID <number>  --from=<ledger address> --useLedger --ledgerAddresses=5
-```
+Each of these elements is important for navigating the governance calls effectively and maximizing the chance of your proposal's success.
+
+### **FAQ Section for Celo Governance**
+
+1. **What is a multisig wallet, and why is it recommended for proposals?**
+    - A multisig wallet requires multiple signatures to authorize transactions, providing increased security and trust for proposals involving treasury funds.
+2. **Who are the Governance approvers?**
+    - Governance approvers are individuals or entities with the authority to review and approve proposals before they go to a community vote, ensuring they meet certain criteria and standards.
+3. **Who are the CGP editors?**
+    - CGP (Celo Governance Proposal) editors are responsible for reviewing and managing the content of governance proposals submitted on GitHub to ensure clarity, completeness, and adherence to the format.
+4. **How to reach CGP editors?**
+    - To reach CGP editors, you can use the Celo Discord channel. They are available to assist with questions and provide guidance on the proposal process.
+5. **Can I submit the proposal again?**
+    - If a proposal is rejected or needs significant revisions, it may be resubmitted after addressing the community's feedback and making necessary adjustments.
+6. **What if I made a mistake in the proposal?**
+    - If you discover a mistake in your proposal, it's important to communicate this to the community and CGP editors as soon as possible. Depending on the stage of the proposal, you may need to withdraw and resubmit it with corrections.
+
+## How to create Multisig with Safe and withdraw fund from Treasury?
+
+### **Step: 1 Log in to your Safe**
+
+Log in and click `New Transaction` on the left hand side of the page. This wil bring up a modal, click `Contract Interation`.
+
+![Governance Safe Creation process step 1](/img/doc-images/governance/governance-safe-multisig-1.png)
```

Comment:
Can we replace all screenshots with the ones from the official Safe website?

Reply:
@viral-sangani can you add these changes?

Reply:
@viral-sangani bump